app/views/dashboard/giveaways/show.html.erb

```diff
@@ -0,0 +1,34 @@
+<div class="container">
+  <div class="columns is-multiline">
+    <div class="column is-6 is-offset-3"><div class="box <%= @giveaway.approved_to.nil? ? 'has-background-white-ter' : 'has-background-primary' %>">
+      <div class="card">
+        <div class="card-content">
+          <div class="content">
+            <% unless @giveaway.approved_to.nil? %>
+              <p class="has-text-right">Approved</p><br>
+            <% end %>
+            <small><%= l(@giveaway[:created_at], format: :short) %></small>
+            <p><code class="html"><%= @giveaway.location %></code></p>
+            <h2><%= @giveaway.title %></h2>
+            <p><%= @giveaway.description %></p>
+            <small><p><%= t('.contact') %>: <%= link_to @giveaway.user.name.capitalize, user_path(id: @giveaway.user_id) %></br> 
+            
+            <% unless @giveaway.pictures.empty? %>
+              <%= render 'pictures/picture'%>
+            <% else %>
+              <div class="image">
+                no images
+              </div>
+            <% end %>
+            <%= render 'footer' %><br>
+            <%= render 'claimed_by' %>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+<% @comments.each do |comment| %>
+  <%= render comment %>
```

Comment:
I belive here you can do: `render partial: 'comment', collection: @comments, as: :comment`

Reply:
@violeta-p Isn't that long version of the render call in line 32? Should I change it?